src/release/main.d

```diff
@@ -335,6 +332,28 @@ ActionList prepareMinorRelease ( ref HTTPConnection con, ref Repository repo,
 
 /*******************************************************************************
 
+    Prepares the actions to clar the release notes
+
+    Returns:
+        actions required to clear the release notes
+
+*******************************************************************************/
+
+ActionList clearReleaseNotes ( )
```

Comment:
I don't think it's a good approach.. that way everyone who wants to know have to follow tons of redirections to find out what exactly has been changed